Formula/vivid.rb

```diff
@@ -0,0 +1,17 @@
+class Vivid < Formula
+  desc "Generator for LS_COLORS with support for multiple color themes"
+  homepage "https://github.com/sharkdp/vivid"
+  url "https://github.com/sharkdp/vivid/archive/v0.6.0.tar.gz"
+  sha256 "c8640f524aef1cd4dc15286bdc6189894ad067ea79bf8c40b9ca8d9d752d161f"
+  license "MIT"
```

Comment:
Should be `license any_of: ["MIT", "Apache-2.0"]`

Reply:
👍

Reply:
Done.